hack/lib/golang.sh

```diff
@@ -397,7 +397,7 @@ kube::golang::build_binaries_for_platform() {
         "${nonstatics[@]:+${nonstatics[@]}}"
     fi
     if [[ "${#statics[@]}" != 0 ]]; then
-      CGO_ENABLED=0 go install -installsuffix cgo "${goflags[@]:+${goflags[@]}}" \
+      CGO_ENABLED=1 go install -installsuffix cgo "${goflags[@]:+${goflags[@]}}" \
```

Comment:
https://github.com/kubernetes/kubernetes/commit/9a901ed46db225f588cc691feeb47e7ace92bbc8 explicitly made kube-apiserver a static binary. If we want to reverse that to add a cgo dependency, we'll need broader agreement